python/doc/examples/probabilistic_modeling/create_stationary_covmodel.ipynb

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
The Expectation notation is not clear: it should be E.
The matrix A is not defined in the text.